src/fetchHandler.js

```diff
@@ -25,6 +25,19 @@ let fetchHandler = {
     .then(response => response.json())
     .then(data => data.destinations)
     .catch(error => console.log('error', error))
+  },
+
+  fetchPostTrip(newTripData) {
+    return fetch('https://fe-apps.herokuapp.com/api/v1/travel-tracker/data/trips/trips', {
+      method: 'POST',
+      headers: {
+        "Content-Type": "application/json"
+      },
+      body: JSON.stringify(newTripData)
+    })
+    .then(response => response.json())
+    .then(data => console.log('good', data, newTripData))
```

Comment:
Great API request methods! On line 39 you might want to return instead of console log it??